guild/tests/run-attrs.md

```diff
@@ -42,7 +42,7 @@ Non-core attributes, however, are written.
     marked: no
     label:
     sourcecode_digest: ...
-    vcs_commit:
+    vcs_commit:...
```

Comment:
I don't understand why this addition came about. I'll investigate it once the rest of the errors are worked out.